test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -782,6 +782,69 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		})
 	})
 
+	Describe("with a windows agent pool", func() {
+		It("kubelet service should be able to recover when the docker service is stopped", func() {
+			if eng.HasWindowsAgents() {
+				kubeConfig, err := GetConfig()
+				Expect(err).NotTo(HaveOccurred())
+				master := fmt.Sprintf("%s@%s", eng.ExpandedDefinition.Properties.LinuxProfile.AdminUsername, kubeConfig.GetServerName())
+				nodeList, err := node.GetReady()
+				Expect(err).NotTo(HaveOccurred())
+
+				simulateDockerdCrashScript := "simulate-dockerd-crash.cmd"
```

Comment:
rather than scp & ssh, I think it might be cleaner to just run `ssh ... cmd /c 'net stop docker & ping localhost -t 1 -w 5 & net start docker'`

Reply:
I'll go ahead and do that in a followup PR since this is passing though

Reply:
@PatrickLang was just going to have a look a this comment. But I see the PR is already merged 👍 

Not sure if `ssh ... cmd /c 'net stop docker & ping localhost -t 1 -w 5 & net start docker'` will do the trick though. The situation that I wanted to create is:
1. `docker` is stopped
2. `kubelet` is restarted

That is what caused the original issue at least. I did notice during testing that the `kubelet` itself (not the service but the executable) does not crash/exit in reasonable amount of time when you crash the docker service (`taskkill /F /FI "SERVICES eq yourservice"`). That is why the `kubelet` is restarted over SSH, in the end the goal was to verify that the `kubelet` service could start when the `docker` service was stopped.